test/type/parameterized_existential.swift

```diff
@@ -91,3 +91,9 @@ func protocolCompositionNotSupported1(_: SomeProto & Sequence<Int>) {}
 
 func protocolCompositionNotSupported2(_: any SomeProto & Sequence<Int>) {}
 // expected-error@-1 {{non-protocol, non-class type 'Sequence<Int>' cannot be used within a protocol-constrained type}}
+
+func increment(n : any Sequence<Float>) {
```

Comment:
Can you also test the case where `func` is generic and the argument to the existential contains a generic parameter type?